docs/CHILI-GraFx/guides/setup-fsso/index.md

```diff
@@ -1,10 +1,10 @@
-# Setup Federated Single sign on
+# Setup Federated Single Sign On
```

Comment:
the page is titled "Setup FSSO" but it doesn't do any setup - title is bit misleading
1. I think it should be a parent page for how-to/FSSO and as child pages it should have different standards (OIDC, SAML) and examples
2. in this page itself, it should have a list of standards with examples to them ex.
``` 
1. Preparation
Please choose standard
Please contact Chili CS with your chosen standard - they will give you setup information based on standard 
2. 
Do setup on your site
Setup steps
* SAML(link) - examples EntraID(link)
* OIDC(link) - examples xxx(link)
3. Verify SSO connection

Reply:
Reorganized this:
![image](https://github.com/chili-publish/grafx-documentation/assets/3585108/fd4b2d30-1e6e-4dc5-99b6-61732d5ad2e5)